rma_sale/wizards/rma_order_line_make_sale_order_view.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 Eficent Business and IT Consulting Services S.L.
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0) -->
+<openerp>
```

Comment:
Please Use `odoo` instead of `openerp` and also remove 'data' in all .xml file if no need.